mediator/src/controllers/openmrs.ts

```diff
@@ -0,0 +1,15 @@
+import { logger } from '../../logger';
+import { syncPatients, syncEncounters } from '../utils/openmrs_sync'
+
+export async function sync() {
+  try {
+    const startTime = new Date();
+    startTime.setHours(startTime.getHours() - 1);
+    await syncPatients(startTime);
+    await syncEncounters(startTime);
```

Comment:
If these two requests can run in parallel we can wrap them using `promise.all`

Reply:
they cannot run in parallel; encounters reference patients and when a patient is created with an encounter in the same sync, the patient needs to be created first